tap_google_analytics/ga_client.py

```diff
@@ -181,17 +183,25 @@ def lookup_data_type(self, type, attribute):
     def process_stream(self, stream):
         try:
             records = []
-            report_definition = self.generate_report_definition(stream)
-            nextPageToken = None
+            start_date = datetime.strptime(self.start_date, "%Y-%m-%d")
+            end_date = start_date + timedelta(days=1)
+
+            while end_date.date() <= datetime.now().date():
```

Comment:
@vitaliizazmic is the request to google using the dates for anything? where is the chunked sync actually happening? this seems like we're making the same request over and over again